js/packages/cli/src/candy-machine-cli.ts

```diff
@@ -291,14 +291,19 @@ programCommand('verify_token_metadata')
     const { number } = cmd.opts();
 
     const startMs = Date.now();
-    log.info('started at: ' + startMs.toString());
+    log.info(
```

Comment:
This does quite a bit of work to generate the log message and should be inside something like 
```
if (log.info.enabled) {
 ...
}
```
To avoid doing that when we're not logging.